salt/orch/update.sls

```diff
@@ -241,6 +251,16 @@ pre-orchestration-migration:
     - require:
       - salt: {{ worker_id }}-update-post-start-services
 
+# Remove the now defuct caasp_fqdn grain
+{{ worker_id }}-remove-caasp-fqdn-grain:
+  salt.function:
+    - tgt: {{ worker_id }}
+    - name: grains.remove
```

Comment:
There's no `grains.remove` state but `grains.absent`, also with `absent` I think we want to use `destructive: True`.

Reply:
Or `grains.remove` (https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.grains.html#salt.modules.grains.remove) also expects the value to be passed.